File: WebLab3/src/main/java/com/arslanka/backend/models/daos/TableRow.java
package com.arslanka.backend.models.daos;

import java.io.Serializable;
import java.time.OffsetDateTime;

import jakarta.enterprise.context.RequestScoped;
import jakarta.inject.Named;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Named("tableRow")
@RequestScoped
@Data
@AllArgsConstructor
@NoArgsConstructor
public class TableRow implements Serializable {
    private Long id;
    private Point2D point2D;
    private Double radius;
    private OffsetDateTime requestTime;
    private Boolean hit;
}

File: web-basics/WebLab2/src/main/kotlin/com/arslanka/weblab2/models/dao/TableRow.kt
package com.arslanka.weblab2.models.dao

import java.math.BigDecimal
import java.time.LocalDateTime

data class TableRow(
    val xCoor: BigDecimal,
    val yCoor: BigDecimal,
    val radius: BigDecimal,
    val isHit: Boolean,
    val localTime: LocalDateTime,
)